CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.4.10 (2023-08-04)
+### Added
+- Initialise BespotSDK when we turn on the network by running the application initially without network.
```

Comment:
Update the description here. To match the one of the private repo (see comment [here](https://github.com/bespot/bespot-sdk-ios/pull/64#discussion_r1293355212)) but without the `SDK: ` prefix. 
-> _"Complete BespotSDK initialisation (authenticate with server) when network is not available initially but becomes available later on."_